subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/Project.java

```diff
@@ -242,6 +259,63 @@ private void addLinkedResourcesToXml() {
         }
     }
 
+    private void addResourceFiltersToXml() {
+        // other nodes like linkedResources are appended even if they are empty, but as
```

Comment:
Ok, I will update the old failing tests that were not expecting this empty node.